docs/track.rst

```diff
@@ -70,7 +70,7 @@ If you have added a remote you should also add it in ``~/.rally/rally.ini``, oth
 
     private.url = <<URL_TO_YOUR_ORIGIN>>
 
-Rally will then automatically update the local tracking branches before the benchmark starts.
+If you specify ``--track-repository=private``, Rally will check whether there is a directory ``~/.rally/benchmarks/tracks/private``. If there is none, it will use the provided URL to clone the repo. However, if the directory already exists the property is irrelevant and Rally will just update the local tracking branches before the benchmark starts.
```

Comment:
Maybe replace `if the directory already exists the property is irrelevant` with `if the directory already exists, the property gets ignored`

Reply:
Thanks for the suggestion; makes sense to me. I'll change the wording before merging.